x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/utils/get_engines_display_text.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { ADMIN, ALL } from '../constants';
+import { IApiToken } from '../types';
+
+export const getEnginesDisplayText = (apiToken: IApiToken): JSX.Element | string => {
+  const { type, access_all_engines: accessAll, engines = [] } = apiToken;
+  const engineList = () => (
+    <ul className="credentials-engine-list">
+      {engines.map((engine) => (
+        <li key={engine} className="credentials-engine-list__engine">
```

Comment:
We can lose these CSS classes, we don't appear to be using them

```suggestion
    <ul>
      {engines.map((engine) => (
        <li key={engine}>
```

Reply:
9e6b0c4